ruby/2.5.7/Dockerfile

```diff
@@ -0,0 +1,103 @@
+FROM socrata/base
+MAINTAINER Socrata <sysadmin@socrata.com>
+
+# skip installing gem documentation
+RUN mkdir -p /usr/local/etc \
+	&& { \
+		echo 'install: --no-document'; \
+		echo 'update: --no-document'; \
+	} >> /usr/local/etc/gemrc
+
+ENV RUBY_MAJOR 2.5
+ENV RUBY_VERSION 2.5.7
+ENV RUBY_DOWNLOAD_SHA256 201870e8f58957d542233fef588b1d76f7bf962fea44dcbd2237f4a5899a3f95
+ENV RUBYGEMS_VERSION 2.7.6
+ENV BUNDLER_VERSION 1.17
```

Comment:
we should probably include logic to install both 1.x and 2.x versions of bundler in the files where we are installing ruby < 2.6. I believe 2.6+ versions come with bundler 2.0.x included.

Reply:
I just installed `2.6.5` via rbenv and it looks like it installed `1.17.2`.